src/main/webapp/app/assessment-instructions/expandable-problem-statement/expandable-problem-statement.component.ts

```diff
@@ -0,0 +1,26 @@
+import { Component, OnInit, Input } from '@angular/core';
+import { ProgrammingExercise } from 'app/entities/programming-exercise';
+import { SafeHtml } from '@angular/platform-browser';
+import { ArtemisMarkdown } from 'app/components/util/markdown.service';
+import { Exercise, ExerciseType } from 'app/entities/exercise';
+
+@Component({
+    selector: 'jhi-expandable-problem-statement',
+    templateUrl: './expandable-problem-statement.component.html',
+    styleUrls: ['../assessment-instructions.scss'],
+})
+export class ExpandableProblemStatementComponent implements OnInit {
+    @Input() exercise: ProgrammingExercise;
```

Comment:
I think you should have `Exercise` type since we are passing different types of exercise here

Reply:
No, we are only passing programming exercises here, because only programming exercises problem statements are rendered differently. Other exercises are passed in assessment-instruction-component :)

Reply:
I think @maxr96 is right.
The (Modeling) Exercise passed to the assessment-instruction-component is lated passed into this one.